evalml/tests/automl_tests/test_automl_search_classification.py

```diff
@@ -482,6 +482,7 @@ def test_describe_pipeline_objective_ordered(X_y_binary, caplog):
         problem_type="binary",
         objective="AUC",
         max_iterations=2,
+        optimize_thresholds=False,
```

Comment:
Why set these to `False`?

If you did it to avoid having the tests waste time calling the threshold optimizer, then, lol I agree we shouldn't run the optimizer in every test. I do wonder if there's a different way to accomplish this though. Could we mock `BinaryClassificationObjective.optimize_threshold` instead, in the same way we mock pipeline `fit` and `score` in many tests?

Reply:
Removed the param here!